pkg/pillar/cmd/zedrouter/dnsmasq.go

```diff
@@ -273,9 +273,9 @@ func createDnsmasqConfiglet(
 		dhcpRange = netconf.DhcpRange.Start.String()
 	}
 	if isIPv6 {
-		file.WriteString(fmt.Sprintf("dhcp-range=::,static,0,60m\n"))
+		file.WriteString(fmt.Sprintf("dhcp-range=::,static,0,12h\n"))
```

Comment:
Hey @eriknordmark @gkodali-zededa -- what's the downside of bumping it to 12h?

Reply:
Since, we changed udhcpc_script.sh to not touch interface IP/mask unless there is change, this change may not be required. IP/mask that we make dnsmasq to offer is static, and the app/container should always receive the same IP/mask with each DHCP renewal.
One side effect of having longer lease time will be the zombie leases on devices in test setups, where we keep adding/removing apps at high frequency. At the same time, we have also patched dnsmasq to force clear leases when we decide to re-use IP addresses that are part of existing leases. So, the effect should be minimal in this case.

Reply:
@adarsh-zededa can you check if the issue is gone with @gkodali-zededa 's fix to the script?